fivetran/connectors/lambda: Improve and extract lambda module (#137)
dbl-works/terraform: iam/iam-policy-for-fivetran-lambda/README.md

## IAM for fivetran lambda

> Creates a IAM role which connect lambda and fivetran

Reviewer: ```suggestion
Creates an IAM role that connects Lambda and Fivetran.
```
